docs/tutorial/introduction.rst

```diff
@@ -20,43 +20,68 @@ Required software:
+
+These algorithms allow researchers to systematically propose hypothetical subnetworks that capture the unique characteristics of a given context without having to experimentally test every individual interaction.
+
+Running multiple pathway reconstruction algorithms on several datasets is difficult due to each requiring unique inputs, software environments, and some are not readily available.
+To perform post-analysis, most algorithm outputs must first be converted into a unified, standardized format.
 
 What is SPRAS?
 ===============
-The Signaling Pathway Reconstruction Analysis Streamliner (SPRAS) is a computational framework that unifies, standardizes, and streamlines the use of diverse pathway reconstructon algorithms.
 
-SPRAS provides an abstraction layer for pathway reconstruction algorithms by organizing every step into a unified schema. It uses workflow management (Snakemake), containerization, and config-driven runs to build modular and interoperable pipelines that cover the entire process:
+.. image:: ../_static/spras-overview.png
+   :alt: spras overview
+   :width: 600
+   :align: center
+
+.. raw:: html
+
+   <div style="margin:20px 0;"></div>
+
+Signaling Pathway Reconstruction Analysis Streamliner (SPRAS) is a computational framework that unifies and simplifies the use of diverse pathway reconstruction algorithms.
+
+SPRAS allows users to run multiple datasets across multiple algorithms and many parameter settings in a single workflow.
+The framework automatically handles data preprocessing, algorithm execution, and post-processing, allowing users to run multiple algorithms seamlessly without manual setup.
+Built-in analysis tools enable users to explore, compare, and evaluate reconstructed pathways with ease.
+
+SPRAS is implemented in Python and leverages two technologies for workflow automation:
+
+- Snakemake: a workflow management system that defines and executes jobs automatically, removing the need for users to write complex scripts
+- Docker: runs algorithms and post analysis in a containerized environment.
 
-1. Pre-processing of data
-2. Algorithm execution
-3. Post-processing of results
-4. Downstream analysis and evaluation
 
-A key strength of SPRAS is automation. From user provided input data and configurations, it can generate and execute complete workflows without requiring users to write complex scripts. This lowers the barrier to entry, allowing researchers to apply, evaluate, and compare multiple pathway reconstruction algorithms without deep computational expertise.
 
-SPRAS also supports scalable analyses, making it especially valuable for a large number of datasets and systematic investigations. In addition, it provides built-in evaluation and post analysis tools that provide further insights of the algorithm outputs.
```

Comment:
Was this text not important? Some of it may be useful to leave in writing even if you don't read it during the tutorial.

Reply:
I took it out because I wasn't planning on saying it during the tutorial.